src/cbmc/cbmc_solvers.cpp

```diff
@@ -26,18 +20,11 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "counterexample_beautification.h"
 #include "version.h"
 
-/*******************************************************************\
-
-Function: cbmc_solverst::get_smt1_solver_type
-
-  Inputs: None
-
- Outputs: An smt1_dect::solvert giving the solver to use.
-
- Purpose: Uses the options to pick an SMT 1.2 solver
-
-\*******************************************************************/
-
+/// Uses the options to pick an SMT 1.2 solver
+///
+/// parameters: None
```

Comment:
I suppose in this case we don't really want this line, perhaps we can special case `Parameters: None` (or even Parameters: followed by a single word) to mean no generated parameters.